doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1858,6 +1858,31 @@ JSON files for `Creative Commons licenses <https://creativecommons.org/about/ccl
 
 .. _adding-custom-licenses:
 
+Adding Software Licenses
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+JSON files for software licenses are provided below.
+
+- :download:`licenseMIT.json <../../../../scripts/api/data/licenses/licenseMIT.json>`
+- :download:`licenseApache-2.0.json <../../../../scripts/api/data/licenses/licenseApache-2.0.json>`
+
+Contributing to the Collection of Standard Licenses Above
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not find the license JSON you need above, you are encouraged to contribute it to this documentation. Following the Dataverse 6.2 release, we have standardized on the following procedure:
+
+- Look for the license at https://spdx.org/licenses/
+- ``cd scripts/api/data/licenses``
+- Copy an existing license as a starting point.
+- Name your file using the SPDX identifier. For example, if the identifier is ``Apache-2.0``, you should name your file ``licenseApache-2.0.json``.
+- For the ``name`` field, use the "short identifier" from the SPDX landing page (e.g. ``Apache-2.0``).
+- For the ``description`` field, use the "full name" from the SPDX landing page (e.g. ``Apache License 2.0``).
+- For the ``uri`` field, we encourage you to use the same resource that DataCite uses, most of the times this will be the same than the first provided resource on SPDX, but please get in contact with us if you have any questions on the cominity group.
```

Comment:
Right now the new licenses don't strictly follow this advice. The easiest change might be to just use the DataCite URIs for both Apache 2 and MIT licenses below. Alternately, if there's interest in using other forms (e.g. using https for the Apache URL and license/MIT - see below) perhaps this could be softened: 
we encourage you to use the same resource that DataCite uses, which is often the same as the first "Other web pages for this license" on the SPDX page for the license. When these differ, or there are other concerns about the URI DataCite uses, please reach out to the community to see if a consensus can be reached.